packages/server/src/lib/api-client.ts

```diff
@@ -16,6 +17,7 @@ export class FormServiceClient implements FormService {
   constructor(private ctx: FormServiceClientContext) {}
 
   async addForm(form: Blueprint) {
+    console.log('add form');
```

Comment:
Can we remove or are there cases we still need to log to see if this method is called?